### PR TITLE
Added Mozilla Thunderbird version 91.5.1.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/42124)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+- InstallerLocale: en-US
+  InstallerType: exe
+  Architecture: x86
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win32/en-US/Thunderbird%20Setup%2091.5.1.exe
+  InstallerSha256: E5C89B5BA9B4D951C72216AF8409EA60BE8268C530DCA31E8866C90E4BD4211D
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: en-GB
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:46]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+- InstallerLocale: en-GB
+  InstallerType: exe
+  Architecture: x86
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win32/en-GB/Thunderbird%20Setup%2091.5.1.exe
+  InstallerSha256: 6D952CA583FC7A618E60F845AB9CF77A292D7D844111D3C33BF16D8746AE3537
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: de-DE
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:74]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  Architecture: x64
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win64/de/Thunderbird%20Setup%2091.5.1.msi
+  InstallerSha256: 1E6DD53CCB3F23599D827BCD74D5D375FED43D6B63FA3F53AF1087E963D35814
+  ProductCode: "{18D4D2F4-B394-48A2-8886-9C68180CEAD5}"
+- InstallerLocale: de-DE
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:80]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+- InstallerLocale: de-DE
+  InstallerType: exe
+  Architecture: x86
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win32/de/Thunderbird%20Setup%2091.5.1.exe
+  InstallerSha256: 50B535C3863828C625457F761DAC9F2D5E311319DDCD0944C0E7C5499344C689
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: zh-CN
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:102]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+  Architecture: x64
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win64/zh-CN/Thunderbird%20Setup%2091.5.1.msi
+  InstallerSha256: A6F6AABE32369F03824B95DEC6CFD980D0964C1D8BF8532DE47B2272F199C23E
+  ProductCode: "{18D4D2F4-B394-48A2-8886-9C68180CEAD5}"
+- InstallerLocale: zh-CN
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:108]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+- InstallerLocale: zh-CN
+  InstallerType: exe
+  Architecture: x86
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win32/zh-CN/Thunderbird%20Setup%2091.5.1.exe
+  InstallerSha256: DE43298ACC5C2296EE4F9579679B9AE077CEE519CB1D92812FEE519BE6DEE74A
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: zh-TW
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:130]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+- InstallerLocale: zh-TW
+  InstallerType: exe
+  Architecture: x86
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win32/zh-TW/Thunderbird%20Setup%2091.5.1.exe
+  InstallerSha256: 53E899C1B3500627A4F37F15747577EA85A1E2AE4BA34BDFFE1727DF5D7529DC
+  InstallerSwitches:
+    Silent: -ms
+    SilentWithProgress: -ms
+- InstallerLocale: es-ES
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:158]:
```suggestion
  InstallerType: wix
```

File: manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+  Architecture: x64
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.5.1/win64/es-ES/Thunderbird%20Setup%2091.5.1.msi
+  InstallerSha256: A7CA6EB6F1FAFC32136A1A10A075CAA9B63283EDD3343634712B6556AE3F6B78
+  ProductCode: "{18D4D2F4-B394-48A2-8886-9C68180CEAD5}"
+- InstallerLocale: es-ES
+  InstallerType: msi
```
Comment (OfficialEsco) [inline manifests/m/Mozilla/Thunderbird/91.5.1/Mozilla.Thunderbird.installer.yaml:164]:
```suggestion
  InstallerType: wix
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
🚨🚨🚨

**Pull request merged**